### DISCUSSION TITLE
Blank screen after sending outlook invite

### DISCUSSION THREAD

**Discussion by shaibebe** [InteractionDesignFoundation/add-event-to-calendar-docs#66] [General]
Hi, I try using the outlook deeplink, it's actually worked.
But after is press on "send" button, I got a blank screen.
It happened also in Live or 365 version.
It's look like the same link if I used the outlook send invitation button or the deeplink button, but only the deeplink got the blank screen.
Somebody knows how to fix it?

**Comment by a-taltal** [comment]
@shaibebe  did you find a solution?

**Comment by shaibebe** [comment]
@a-taltal unfortunately not.

**Comment by Pjaerr** [comment]
👋 Sorry just noticed this in my emails @shaibebe ...I had this issue with the blank screen showing after sending...using a deeplink like: `https://outlook.office.com/owa/?path=/calendar/action/compose` does not have this issue. 

It's been a while since I worked with this so I can't remember if there were any caveats but maybe it'll help point you in the right direction. (worth noting that I found this way by just messing around with the url, so it is very _unofficial_ but I've been using it for over a year with no issues)

**Comment by shaibebe** [comment]
@Pjaerr thank you. I will check it.
I see that is for 365.
Do you have the link for outlook live?

**Comment by Pjaerr** [comment]
@shaibebe I might be mistaken but I believe it works for live too.   (if not, then maybe just replacing `office` with `live` might work?). Hopefully you can get it to work for you either way 🤞
